src/PaymentGateways/PayPalCommerce/ScriptLoader.php

```diff
@@ -208,15 +203,13 @@ public function loadPublicAssets()
     /**
      * Get PayPal partner js url.
      *
+     * @unreleased sandbox PayPal partner js loads slow. So we are using live url for now.
```

Comment:
Is it in any way a problem that we're not using the sandbox partner js?

Reply:
@JasonTheAdams It is not a problem. Paypal loads the same SDK in the sandbox and the live mode. Js SDK loads slowly in sandbox mode, so I used only the live mode JS. We can switch it manually if we need debugging.